blob/azureblob/azureblob.go

```diff
@@ -17,33 +17,11 @@
 //
 // Open URLs
 //
-// For blob.Open URLs, azureblob registers for the scheme "azblob"; URLs start
-// with "azblob://".
-//
-// The URL's Host is used as the bucket name.
-//
-// By default, credentials are retrieved from the environment variables
-// AZURE_STORAGE_ACCOUNT, AZURE_STORAGE_KEY, and AZURE_STORAGE_SAS_TOKEN.
-// AZURE_STORAGE_ACCOUNT is required, along with one of the other two. See
+// For blob.OpenBucket URLs, azureblob obtains credentials from the environment
+// variables AZURE_STORAGE_ACCOUNT, AZURE_STORAGE_KEY, and
+// AZURE_STORAGE_SAS_TOKEN. AZURE_STORAGE_ACCOUNT is required, along with one of
+// the other two. For more on SAS tokens, see
```

Comment:
Can you add back something to the comment with:

1. What the scheme is.
2. A sample URL.

?